src/ecwam/CMakeLists.txt

```
@@ -483,7 +483,11 @@ foreach( prec sp dp )
     endif()
 
     if( HAVE_CUDA )
```

```suggestion
    if( HAVE_CUDA AND CMAKE_Fortran_COMPILER_ID MATCHES NVHPC )
```
Checking for NVHPC compiler just to be sure